tests/system/feature_store/test_feature_store.py

```diff
@@ -406,20 +406,22 @@ def test_get_offline_features_with_or_without_indexes(self):
         ), "'another_time' column should be present"
 
         # with_indexes = False, entity_timestamp_column = "invalid" - should return the timestamp column
-        df_with_time = fstore.get_offline_features(
-            vector, entity_timestamp_column="another_time"
-        ).to_dataframe()
+        df_without_time_and_indexes = fstore.get_offline_features(vector).to_dataframe()
         assert isinstance(
-            df_with_time.index, pd.core.indexes.range.RangeIndex
+            df_without_time_and_indexes.index, pd.core.indexes.range.RangeIndex
+        ), "index column is not of default RangeIndex"
```

Comment:
I really think these error messages are unnecessary. When an assertion fails, you will see the condition that it failed on, and even the values involved.

Reply:
@gtopper so to delete all of them in this test? I also think it unnecessary.

Reply:
Sure, that would be great.